pkg/apis/kops/cluster.go

```diff
@@ -276,12 +276,26 @@ type KubeDNSConfig struct {
 	ServerIP string `json:"serverIP,omitempty"`
 }
 
+// StorageType describes storage type for an etcd cluster (etcd2, etcd3)
+type StorageType string
+
+const (
```

Comment:
I think this maps to `--storage-type` which maps to the storage driver flag to apiserver.  The image is actually separate, and is determined by `version` in EtcdCluster.